packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -637,6 +639,8 @@ class _IOSSimulatorLogReader extends DeviceLogReader {
     _systemProcess.stdout.transform(UTF8.decoder).transform(const LineSplitter()).listen(_onSystemLine);
     _systemProcess.stderr.transform(UTF8.decoder).transform(const LineSplitter()).listen(_onSystemLine);
 
+    // Callback hookups' Futures are uninteresting.
+    // ignore: unawaited_futures
     _deviceProcess.exitCode.whenComplete(() {
```

Comment:
Yeah, we don't want to `await` here as the `_deviceProcess` will outlive the `_start()` method. (and, we don't really care about about the specific reason the tail process exits, we just want to clean up the lines controller when the tail process completes).

We may want to catch and ignore any exceptions from `_deviceProcess` however, so they don't bubble through to the zone handler.